source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -371,6 +373,158 @@ namespace
         return prim;
     }
 
+    PvtPrim* readCollection(const CollectionPtr& src, PvtPrim* parent, PvtStage* stage)
+    {
+        const RtToken name(src->getName());
+
+        PvtPrim* prim = stage->createPrim(parent->getPath(), name, RtCollection::typeName());
+        RtCollection collection(prim->hnd());
+        collection.getIncludeGeom().setValueString(src->getIncludeGeom());
+        collection.getExcludeGeom().setValueString(src->getExcludeGeom());
+
+        // createCollectionConnections() handles creating collection inclusion
+        // connections.
+        return prim;
+    }
+
+    void createCollectionConnections(const vector<CollectionPtr>& collectionElements, PvtPrim* parent)
```

Comment:
The name of this function seems misleading to me, but maybe that's just me.

Reply:
Up for suggestions. It's making connections between collections that have "include" relationships.

Reply:
You seem to use this create...Connections pattern throughout, not a big deal I guess.